cellpack/autopack/BaseGrid.py

```diff
@@ -886,74 +885,6 @@ def computeVolume(self, space=None, fbox_bb=None):
             totalVolume = V * unitVol
         return totalVolume
 
-    def create_rapid_model(self):
-        self.rapid_model = RAPIDlib.RAPID_model()
-        # need triangle and vertices
-        self.rapid_model.addTriangles(
-            numpy.array(self.vertices, "f"), numpy.array(self.faces, "i")
-        )
-
-    def get_rapid_model(self):
-        if self.rapid_model is None:
-            self.create_rapid_model()
-        return self.rapid_model
-
-    def one_rapid_ray(self, pt1, pt2, diag):
-        # return number of triangle /triangle contact
-        helper = autopack.helper
-        rm = self.get_rapid_model()
-        v1 = numpy.array(pt1)
-        direction = helper.unit_vector(pt2 - pt1) * diag
-        v2 = v1 + direction
-        if sum(v1) == 0.0:
-            v3 = v2 + numpy.array([0.0, 1.0, 0.0])
-        else:
-            v3 = v2 + helper.unit_vector(numpy.cross(v1, v2))
-        f = [0, 1, 2]
-        ray_model = RAPIDlib.RAPID_model()
-        ray_model.addTriangles(
-            numpy.array([v1, v2, v3], "f"),
-            numpy.array(
-                [f],
-                "i",
-            ),
-        )
-        RAPIDlib.RAPID_Collide_scaled(
-            numpy.identity(3),
-            numpy.array([0.0, 0.0, 0.0], "f"),
-            1.0,
-            rm,
-            numpy.identity(3),
-            numpy.array([0.0, 0.0, 0.0], "f"),
-            1.0,
-            ray_model,
-            RAPIDlib.cvar.RAPID_ALL_CONTACTS,
-        )
-        # could display it ?
-        return RAPIDlib.cvar.RAPID_num_contacts
-
-    def checkPointInside_rapid(self, point, diag, ray=1):
```

Comment:
this function is called in getSurfaceInnerPoints_jordan so it needs to have an alternative or completely remove getSurfaceInnerPoints_jordan.